browser/ui/views/profiles/brave_profile_menu_view.cc

```diff
@@ -11,17 +11,34 @@
 #include "brave/browser/profiles/profile_util.h"
 #include "chrome/app/vector_icons/vector_icons.h"
 #include "chrome/browser/browser_process.h"
-#include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/profiles/profile_attributes_entry.h"
 #include "chrome/browser/profiles/profile_manager.h"
 #include "chrome/browser/profiles/profile_window.h"
+#include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/ui/browser_finder.h"
+#include "chrome/browser/ui/browser_list.h"
 #include "chrome/browser/ui/signin/profile_colors_util.h"
 #include "chrome/grit/generated_resources.h"
 #include "components/vector_icons/vector_icons.h"
 #include "ui/base/l10n/l10n_util.h"
 #include "ui/base/models/image_model.h"
 
+namespace {
+
+void CloseGuestProfileWindows() {
```

Comment:
what do we need this for?

Reply:
Hmmm... I'm afraid I was too narrow-minded and focused only on getting cr90 to build without actually stopping to think whether this is actually needed or not... and on a second thought I think this is probably not needed so reverting this change (and simply dropping the `CloseGuestProfileWindows()` call from `BraveProfileMenuView::OnExitProfileButtonClicked()`) in https://github.com/brave/brave-core/pull/7866/commits/27d2dd9d5f2f0f2d84a12ab4d641eb48bce19a78